cypress/pageObjects/datasetsPage.ts

```diff
@@ -0,0 +1,7 @@
+class DatasetPage {
+  getDataset(name: string) {
+    cy.get(".flex.items-center.relative.w-full.cursor-pointer").contains(name); //"xl_PGP3140_wgs_NIST-4_2"
```

Comment:
1. I don't like this selector, let's add `data-cy` or `data-test` attribute to the component
2. What does this comment mean? It adds no useful information I'd say